docs/nodes/configuration.md

```diff
@@ -249,7 +290,7 @@ persistent-peers-max-dial-period = "0s"
 flush-throttle-timeout = "100ms"
 
 # Maximum size of a message packet payload, in bytes
-max-packet-msg-payload-size = 1024
+max-packet-msg-payload-size = 1400
```

Comment:
why?

Reply:
this was copied from the generated config.toml. Not sure why it increased

Reply:
https://github.com/tendermint/tendermint/commit/ced66e4eb5498014866f8fda242b62fb5b55009b here is the commit. Seems it was because we arbitrarily picked 1024.